datahub-frontend/README.md

```diff
@@ -51,6 +51,8 @@ the application directly from command line after a successful [build](#build):
 cd datahub-frontend/run && ./run-local-frontend
 ```
 
+If you are running the React app locally via `yarn start`, it will be forwarding graphql requests to port `9002`. In order to use `./run-local-frontend` with the React app, change the httpPort value in [./play.gradle](./play.gradle) to `9002` and run `./gradlew :datahub-frontend:build`.
```

Comment:
An alternative that I tend to use is to just change `frontend.env` PORT value to 9002 and then re-run `run-local-frontend`. This allows you to bypass a build!

Reply:
Ah good to know- updated